ceph/ceph_admin/bootstrap.py

```diff
@@ -14,6 +14,12 @@
 class BootstrapMixin:
     """Add bootstrap support to the child class."""
 
+    def get_method(self, key):
```

Comment:
Could we have some documentation about the method?

Reply:
Added doc string